### ISSUE TITLE
Newer Lib

### ISSUE THREAD

**Issue by TacoOblivion** [hrace009/Perfect-World-API-class-examples#1]
Hi, I updated the monkey lib past the version you have.

https://www.nrgs.org/dev/monkeylib/MonkeyLib2019.zip

The major change was from mysql to mysqli, a Markdown system and maybe some other things like Cloudflare support for sessions.

Before you change anything you may want to test that it's compatible. I'm pretty sure it's not different enough to break anything, but who knows. mysqli was real weird to get working properly as I recall, although that was 4 years ago.

Edit: Oh wow, I noticed that this repo has a really old version of the lib. I thought it was a little newer. The version on here is from when I practically first wrote it. I hope the newer lib is still compatible, cause all of my changes were for the best. I may start a repo for the library since I'm using it in a current project.

**Comment by hrace009** [comment]
hi, yes this repo is your first code in RZ last 4 years a go 🤣 . since i never seen this code at GitHub, and i don't know your GitHub ID, so i decide to create it here.

I think i should transfer it to you, if you don't mind.

and i will try to test the new lib, and i think it will compatible.

**Comment by hrace009** [comment]
Sorry very late to respon
has been update 3 years a go >.<
[commit](https://github.com/hrace009/Perfect-World-API-class-examples/commit/f9c46f4fc98e8d2a2b542c1a3eaab6cce5f63b02)

**Comment by TacoOblivion** [comment]
I have an even newer version. The code has been updated significantly. It's now only compatible with php 7.4 and above and fixed a handful of bugs. I'm going to put the repository up soon and I'll link it here. I have a tendency to work on it every time I get a job for building a website. Which just so happens to be right at this moment.

And as a programmer, I've grown so much from several years ago. I teach programming now. I also changed my username.
